pkg/psmdb/mongos.go

```diff
@@ -450,7 +450,7 @@ func MongosServiceSpec(cr *api.PerconaServerMongoDB, podName string) corev1.Serv
 	switch cr.Spec.Sharding.Mongos.Expose.ExposeType {
 	case corev1.ServiceTypeNodePort:
 		spec.Type = corev1.ServiceTypeNodePort
-		spec.ExternalTrafficPolicy = "Local"
+		spec.ExternalTrafficPolicy = "Cluster"
```

Comment:
please leave the old value and set the new value like this:

```
if cr.CompareVersion("1.19.0") >= 0 {
  spec.ExternalTrafficPolicy = "Cluster"
}
```

this new value should be only used starting from 1.19.0